sources/osg/State.js

```diff
@@ -515,8 +515,10 @@ define( [
             // if the the stack has changed but the last applied attribute is the same
             // then we dont need to apply it again
             if ( attributeStack.lastApplied !== attribute ) {
-                gl.activeTexture( gl.TEXTURE0 + textureUnit );
-                attribute.apply( this, textureUnit );
+                if ( !attribute._textureNull ) {
```

Comment:
Adds a methods to Texture attribute like isTextureNull then use this method for the test